packages/core/src/lib/markdown-it/plugins/markdown-it-icons.ts

```diff
@@ -1,30 +1,45 @@
-const octicons = require('@primer/octicons');
+import octicons from '@primer/octicons';
+import { IconName } from '@primer/octicons';
+import cheerio from "cheerio";
 
-module.exports = require('markdown-it-regexp')(
+function getOcticonIcon(iconName: string) {
+    const key: IconName | undefined = (Object.keys(octicons) as Array<IconName>).find(key => key === iconName);
+    if (key === undefined) {
+        return null;
+    }
+    return octicons[key];
```

Comment:
```suggestion
  return octicons[iconName as IconName];
```
or if we need to return `null` instead of undefined:
```suggestion
  return octicons[iconName as IconName] ?? null;
```